src/index.html

```diff
@@ -5,8 +5,375 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container container--catalog">
+      <header class="header">
+        <div>
+          <a class="logo" href="#home">
+            <img src="./images/logo.png" alt="logo">
+          </a>
+        </div>
```

Comment:
div is redundant here
```suggestion
          <a class="logo" href="#home">
            <img src="./images/logo.png" alt="logo">
          </a>
```